inspectors/types/iconnames.h

```diff
@@ -150,7 +150,7 @@ class IconNameTypes
 
         CHORD_SYMBOL = 0xF352,
         GUITAR_BEND = 0xF353,
-        GUITAR_TREMOLO_BAR = 0xF355,
+        GUITAR_TREMOLO_BAR = 0xF35C,
```

Comment:
This file seems to not be used anymore, with `framework/ui/view/iconcodes.h` replacing it, but as long as it exists, it might as well be updated.